apps/builder/app/routes/$.tsx

```diff
@@ -15,13 +15,10 @@ export const loader = async ({ request }: LoaderArgs) => {
   }
 
   const params: Params = {
+    imageBaseUrl: env.IMAGE_BASE_URL,
```

Comment:
Lets add comment above about possible values it can be. Also how it different from assetBaseUrl
As it can be just basePath and can be baseUrl.
Same comments for the IMAGE_BASE_URL
Also lets hardcode default for `IMAGE_BASE_URL` at code if not already

Reply:
Hardcoded for builder but would keep required for saas which use own worker implementation.